### PR TITLE
Add cht.exe 0.6

### PR DESCRIPTION
**Pull request by goostleek**
https://github.com/chubin/cheat.sh#windows-command-line-client

### CODE REVIEW

File: bucket/cht.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://github.com/tpanj/cht.exe",
+    "description": "A command line client for https://cht.sh, working examples about various commands, APIs, ... Unified access to the best community driven documentation repositories of the world.",
+    "license": "GPL-3.0-or-later",
+    "version": "0.6",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/tpanj/cht.exe/archive/v0.6.zip#/bin/cht.exe",
+            "hash": "771061cefff950be8ec3f2987e0b56e8ec20b79c8d451e24e5f89b7f391629df"
+        }
+    },
```
Comment (Ash258) [inline bucket/cht.json]:
```suggestion
    },
    "extract_dir": "cht.exe-0.6\\bin",
```

File: bucket/cht.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/tpanj/cht.exe",
+    "description": "A command line client for https://cht.sh, working examples about various commands, APIs, ... Unified access to the best community driven documentation repositories of the world.",
+    "license": "GPL-3.0-or-later",
+    "version": "0.6",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/tpanj/cht.exe/archive/v0.6.zip#/bin/cht.exe",
```
Comment (Ash258) [inline bucket/cht.json]: That's not how it work.

`#/` is for renaming, not specifying what to extract.

```suggestion
            "url": "https://github.com/tpanj/cht.exe/archive/v0.6.zip",
```

File: bucket/cht.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            "hash": "771061cefff950be8ec3f2987e0b56e8ec20b79c8d451e24e5f89b7f391629df"
+        }
+    },
+    "bin": "cht.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://github.com/tpanj/cht.exe/archive/v$version.zip#/bin/cht.exe"
```
Comment (Ash258) [inline bucket/cht.json]:
```suggestion
                "url": "https://github.com/tpanj/cht.exe/archive/v$version.zip"
```

File: bucket/cht.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            }
+        }
```
Comment (Ash258) [inline bucket/cht.json]:
```suggestion
        },
        "extract_dir": "cht.exe-$version\\bin"
```

**Comment by goostleek** [comment]
All fixes applied @Ash258.

**Comment by r15ch13** [comment]
Cool!

**Comment by chubin** [comment]
very cool!

**Comment by tpanj** [comment]
Awesome, sorry for late merge!

**Pull request merged**